expan/core/statistics.py

```diff
@@ -9,6 +9,13 @@ def _delta_mean(x, y):
 	"Implemented as function to allow calling from bootstrap."
 	return np.nanmean(x) - np.nanmean(y)
 
+def make_delta(assume_normal=True, percentiles=[2.5, 97.5],
```

Comment:
I am a little not satisfied with the name ```make_delta```, because it might cause confusion with ```delta```, which also makes a delta with additional parameters. 
but couldn't think of a better name at the moment too.